amazon-eks-nodegroup.yaml

```diff
@@ -283,31 +276,30 @@ Resources:
       LaunchConfigurationName: !Ref NodeLaunchConfig
       MinSize: !Ref NodeAutoScalingGroupMinSize
       MaxSize: !Ref NodeAutoScalingGroupMaxSize
-      VPCZoneIdentifier:
-        !Ref Subnets
+      VPCZoneIdentifier: !Ref Subnets
       Tags:
-      - Key: Name
-        Value: !Sub "${ClusterName}-${NodeGroupName}-Node"
-        PropagateAtLaunch: 'true'
-      - Key: !Sub 'kubernetes.io/cluster/${ClusterName}'
-        Value: 'owned'
-        PropagateAtLaunch: 'true'
+        - Key: Name
+          Value: !Sub ${ClusterName}-${NodeGroupName}-Node
+          PropagateAtLaunch: true
```

Comment:
I'll have to check if this unquoted boolean is accepted by CloudFormation

Reply:
Please verify, but from my observation it gets parsed, correctly, by CloudFormation's yaml parser. YAML itself considers Booleans to be expressed with true and false (no quotes).